.github/workflows/ci.yml

```
@@ -1,37 +1,49 @@
-          cache: 'npm'
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Run Lint
-        run: npm run lint
-
-      - name: Build project
-        run: npm run build
\ No newline at end of file
+name: Next.js CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)